packages/gatsby-recipes/src/providers/gatsby/shadow-file.js

```diff
@@ -114,22 +115,29 @@ export const plan = async ({ root }, { theme, path: filePath, id }) => {
   // eslint-disable-next-line
   const fullFilePathToShadow = path.join(root, `node_modules`, theme, filePath)
 
-  const newContents = await fs.readFile(fullFilePathToShadow, `utf8`)
+  try {
+    newContents = await fs.readFile(fullFilePathToShadow, `utf8`)
+  } catch (e) {
+    // Good chance this file won't exist yet as the theme's NPM package isn't
```

Comment:
I would emit the error as a `.verbose()` just in case it's something else.

Reply:
👍